main.go

```diff
@@ -14,13 +14,13 @@ func main() {
 	var username = flag.String("username", "", "Registry username")
 	var password = flag.String("password", "", "Registry password")
 	//var lastImages = *flag.Int("keep", 10, "Keep Last n images")
-	//var dryRun = *flag.Bool("dry-run",false,"Print old images, don't remove.")
+	var dryRun = flag.Bool("dry-run", false, "Print old images, don't remove.")
```

Comment:
flag yerine [pflag](https://github.com/spf13/pflag) kullansak nasıl olur?

Reply:
Farkları veya avantajları neler?

Reply:
sonradan pointerdan değere döndürmene gerek kalmıyor kanka alttaki kafada takılabiliyorsun. Hem de örneğin --concurrency=10 şeklinde geçiyorsun -concurrency=10 yerine, POSIX stayla.

```go
var flagvar int
func init() {
    flag.IntVar(&flagvar, "flagname", 1234, "help message for flagname")
}
```